.gitignore

```diff
@@ -123,3 +123,6 @@ dmypy.json
 
 # Yarn cache
 .yarn/
+
+# Generated css files by tailwind
+style/app.css
```

Comment:
@agoose77, this is a pre-built file, not sure why we lost it in the gitignore?

Reply:
@parmentelat this is related to the error that you ran into. The app.css is built on demand, but apparently not being triggered on a pip install. I don't know quite enough about what pip is calling to be helpful though, hopefully we will get this in soon though!